loading rules
[()]
[()]
[(Error (load-ascii GroundingSpace-0x55e30df95128 rules.metta) Exception caught:
RuntimeError: load_ascii: file not found

At:
  /home/me/.local/lib/python3.10/site-packages/hyperon/stdlib.py(199): load_ascii_atom
  /home/me/.local/lib/python3.10/site-packages/hyperon/atoms.py(428): execute
  /home/me/.local/lib/python3.10/site-packages/hyperon/atoms.py(203): _priv_call_execute_on_grounded_atom
  /home/me/.local/lib/python3.10/site-packages/hyperon/runner.py(207): run
  /home/me/.local/lib/python3.10/site-packages/hyperon/metta.py(35): main
  /home/me/.local/bin/metta(8): <module>
)]
real 0.04
user 0.03
sys 0.00
